From a627d989901f1d83be29b142309da002a70d0048 Mon Sep 17 00:00:00 2001
From: Wangming Shao <shaowangming@h-partners.com>
Date: Wed, 19 Oct 2022 12:27:45 +0800
Subject: [PATCH 0992/1646] [Backport] set hisi ptt module defconfig

driver inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I5VGP3

Reference: https://gitee.com/openeuler/kernel/commit/0c620424f565c9dcf5997ddb08c6c104a67a5d73

--------------------------------

Add compilation settings for the hisi_ptt module.

Signed-off-by: Wangming Shao <shaowangming@h-partners.com>
Reviewed-by: Yicong Yang <yangyicong@huawei.com>
Reviewed-by: Chao Liu <liuchao173@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Jialin Zhang <zhangjialin11@huawei.com>

 Conflicts:
	arch/arm64/configs/openeuler_defconfig

Signed-off-by: Jialin Zhang <zhangjialin11@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 8889db033b4a..656ccd43a092 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -6101,6 +6101,7 @@ CONFIG_ARM_SPE_PMU=y
 CONFIG_HISI_PMU=m
 CONFIG_HISI_L3T_PMU=m
 CONFIG_HISI_LPDDRC_PMU=m
+CONFIG_HISI_PTT=m
 CONFIG_HISI_PCIE_PMU=m
 # end of Performance monitor support
 
-- 
2.17.1

